### PR TITLE
UMCO-144/change/Wallee_release_1.0.93

### PR DESCRIPTION
**Pull request by sixer1182**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,20 @@ All notable changes to this project will be documented in this file.
...
+ * Add warning on documentation about manual booking for Unzer secured invoices.
+ * Fix for WPML support.
+
+### Changed
+ * Update Unzer certificate.
+ * Added PHP version compatibility text to Sellxed docs.
+ * Improved screening of error messages, do not display details to end users.
+ * Remove subscription entries from documentations.
+ * Updated Klarna Logos.
+ * Added EtiId for Klarna payments.
```
Comment (Ryouzanpaku) [inline CHANGELOG.md]: Where is that change in the code, I can not find anything that looks like it adresses this change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,20 @@ All notable changes to this project will be documented in this file.
...
 
+## [1.0.93](https://github.com/unzerdev/woocommerce/compare/1.0.85..1.0.93)
+
+### Fixed
+ * Add warning on documentation about manual booking for Unzer secured invoices.
+ * Fix for WPML support.
+
+### Changed
+ * Update Unzer certificate.
+ * Added PHP version compatibility text to Sellxed docs.
```
Comment (Ryouzanpaku) [inline CHANGELOG.md:14]: Documentation change does not seam to be part of this PR.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,20 @@ All notable changes to this project will be documented in this file.
...
+ * Improved screening of error messages, do not display details to end users.
+ * Remove subscription entries from documentations.
```
Comment (Ryouzanpaku) [inline CHANGELOG.md:16]: Documentation change does not seam to be part of this PR.

**Review by Ryouzanpaku — COMMENTED** [review]
I could not find evidence for all changelog entries in the code. See comments.

File: CHANGELOG.md (hunk 2 above)
Comment (sixer1182) [reply CHANGELOG.md:14]: Thanks for the hint. Now they should be.

File: CHANGELOG.md (hunk 1 above)
Comment (sixer1182) [reply CHANGELOG.md]: This was actually copied from wallee changelog but I cannot find the change either.
I remove it.

**Review by Ryouzanpaku — APPROVED** [review]
LGTM!

**Pull request merged**